requirements-freeze.txt

```diff
@@ -1,103 +1,96 @@
+-e git+https://github.com/GSA/ckanext-datajson@6dfe7a908a8da6d38ae5da86179a21f558a5344a#egg=ckanext_datajson
 -e git+https://github.com/GSA/ckanext-extlink@69629853f6c7aa4f94c105c5efd76380d70f0843#egg=ckanext_extlink
 -e git+https://github.com/GSA/ckanext-ga-report@7d32eac6db1bc68dd8f3f96f14c6fd3aeec902a9#egg=ckanext_ga_report
--e git+https://github.com/GSA/ckanext-geodatagov@f28f1751d23eac973d96394a2485ccfbd847135b#egg=ckanext_geodatagov
+-e git+https://github.com/GSA/ckanext-geodatagov@d82eb45067f9b8335b6d767499f218431fb6c215#egg=ckanext_geodatagov
 -e git+https://github.com/GSA/ckanext-googleanalyticsbasic@0b90194d77938e59ecb7d6937c201bd6815447f8#egg=ckanext_googleanalyticsbasic
--e git+https://github.com/GSA/ckanext-harvest@8eaf4e1021019d01b34e8ce58aed327a24f59c96#egg=ckanext_harvest
+-e git+https://github.com/GSA/ckanext-harvest@39e9aba1ed28bad3a139924932463f086bbc1b4a#egg=ckanext_harvest
 -e git+https://github.com/GSA/ckanext-qa@2b3353cb3a36e021de448b7d280165a3dbfcf728#egg=ckanext_qa
 -e git+https://github.com/GSA/ckanext-report@f16607e700db67645734257a0c3ac74f4cbdb718#egg=ckanext_report
 -e git+https://github.com/GSA/ckanext-saml2@0e1f11cb3b211e6d337b86ffe7c07620821f7d7d#egg=ckanext_saml2
 -e git+https://github.com/GSA/ckanext-spatial@2a25f8d60c31add77e155c4136f2c0d4e3b86385#egg=ckanext_spatial
-Click==7.0
-decorator==3.4.0
+decorator==4.4.2
 fanstatic==0.12
-Flask==1.0
-FormEncode==1.2.6
+FormEncode==1.3.1
+funcsigs==1.0.2
 gdata==2.0.18
 Genshi==0.6
 GeoAlchemy==0.7.2
-GeoAlchemy2==0.2.6
+GeoAlchemy2==0.7.0
 google-api-python-client==1.4.1
-html5lib==0.9999999
-httplib2==0.11.3
+html5lib==1.0.1
+httplib2==0.17.1
 idna==2.7
-itsdangerous==1.1.0
 Jinja2==2.6
 json-table-schema==0.2.1
 jsonschema==2.4.0
 kombu==2.5.0
-kombu-sqlalchemy==1.1.0
 LEPL==5.1.3
-lxml==3.5.0
-M2Crypto==0.23.0
-Mako==0.9.0
-MarkupSafe==0.18
-meld3==1.0.2
+lxml==4.5.0
+Mako==1.1.2
+MarkupSafe==1.1.1
 messytables==0.15.2
-nose==1.3.0
+nose==1.3.7
 oauth2client==1.4.12
 ofs==0.4.1
 -e git+https://github.com/GSA/OWSLib.git@71a43028777bc982d993876d925699af830e2839#egg=OWSLib
 Pairtree===0.7.1-T
 passlib==1.6.2
 Paste==1.7.5.1
-PasteDeploy==1.5.0
+PasteDeploy==2.1.0
 PasteScript==1.7.5
-pbr==0.8.2
+pbr==0.11.1
 pika==0.9.8
 ply==3.4
 progressbar==2.3
 psycopg2==2.4.5
-pyasn1==0.4.4
-pyasn1-modules==0.2.2
-Pygments==1.6
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+Pygments==2.5.2
 Pylons==0.9.7
 pyparsing==1.5.6
 -e git+https://github.com/GSA/pysaml2.git@8b605c824ff1d042e9040a9971c28f369197cac2#egg=pysaml2
 python-dateutil==1.5
 python-magic==0.4.12
 python-memcached==1.48
-pytz==2016.1
+pytz==2019.3
 pyutilib.component.core==4.5.3
-PyYAML==5.1
--e git+https://github.com/asl2/PyZ3950.git@c2282c73182cef2beca0f65b1eb7699c9b24512e#egg=PyZ3950
+PyYAML==5.3.1
+-e git+https://github.com/asl2/PyZ3950@c2282c73182cef2beca0f65b1eb7699c9b24512e#egg=PyZ3950
 redis==2.10.1
-repoze.lru==0.6
-repoze.who==1.0.18
+repoze.lru==0.7
+repoze.who==2.0
```

Comment:
This is going to break ckanext-saml2 https://github.com/GSA/catalog-app/issues/78

Reply:
We should just copy the documentation from inventory into readme for catalog and add to this PR...

Reply:
inventory does not have to deal with this repoze.who version issue. Its version stays at 1.0.18 as in the freeze file. Catalog upgrades to to 2.0 at some point during the setup.

Reply:
Oh, good observation. That's because in inventory-app we do it in [`start.sh`](https://github.com/GSA/inventory-app/blob/bbc0380cd916c2ac298d464487ad0921349cc910/start.sh#L51), we can do something similar for catalog-app which will avoid this issue.

Reply:
If we run `pip freeze` after the app is setup, it will still get the 2.0 version regardless of where/how it's installed, right?

Reply:
Depends how you setup the app. I usually do `make clean build requirements` which doesn't run the start.sh script.

Reply:
this commit should resolve the issue. https://github.com/GSA/catalog-app/pull/128/commits/a5b4d5d3a9148766240a428a5ff46aab12d91d5c